ui/src/common/components/CitationSummaryGraph/CitationSummaryGraph.jsx

```diff
@@ -87,19 +155,45 @@ class CitationSummaryGraph extends Component {
                     />
                     <YAxis />
                     <VerticalBarSeries
+                      colorType="literal"
                       data={citeableSeriesData}
-                      color={BLUE}
                       barWidth={BAR_WIDTH}
+                      onValueMouseOver={hoveredBar =>
+                        this.onBarMouseHover({
+                          xValue: hoveredBar.x,
+                          type: CITEABLE_BAR_TYPE,
+                        })
+                      }
+                      onValueClick={bar =>
```

Comment:
can you make those class methods? for example, for this one it would be `onCiteableBarClick`